src/main/java/randoop/util/ClassFileConstants.java

```diff
@@ -100,6 +106,22 @@ public static class ConstantSet {
     /** Values that are non-receiver terms. */
     public Set<Class<?>> classes = new HashSet<>();
 
+    /** Set of all enum constants in a class. */
```

Comment:
Please be more specific than "in a class".  Constant variables defined in the class?  Any constant that is ever used in the class?